### DISCUSSION TITLE
[Added] improved modem/router watchdog tutorial

### DISCUSSION THREAD

**Discussion by iec61131** [arendst/Tasmota#20188] [Ideas]
Thanks @arendst  for this amazing project.
After a many hours spent I finally created a cookbook recipe for how to turn $8 S31s into WifiRebooters. 
During this process I found couple issues which I fixed:
- doc watchdog Rules did not work out of box
- discovered bug of missing unix time which prohibits Tasmato to execute rule without WiFi

Please check my project https://github.com/iec61131/WiFiRebooter 
Feel free to merge any of it in the docs etc.

**Comment by pkkrusty** [comment]
Docs pages are maintained by the community, so things probably won't get changed unless you initiate.
